deel/torchlip/modules/module.py

```diff
@@ -29,16 +29,68 @@
 for condensation and vanilla exportation.
 """
 import abc
-from collections import OrderedDict
 import copy
-import logging
+import warnings
 import math
+from collections import OrderedDict
 from typing import Any
 
 import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.utils.parametrize as parametrize
 from torch.nn import Sequential as TorchSequential
+from torch import reshape
+
+
+def _is_supported_1lip_layer(layer):
+    """Return True if the Keras layer is 1-Lipschitz. Note that in some cases, the layer
+    is 1-Lipschitz for specific set of parameters.
+    """
+    supported_1lip_layers = (
+        torch.nn.Softmax,
+        torch.nn.Flatten,
+        torch.nn.Identity,
+        torch.nn.ReLU,
+        torch.nn.Sigmoid,
+        torch.nn.Tanh,
+        Reshape,
```

Comment:
add torch.view to the list ?

Reply:
torch.tensor.view is not a layer

Reply:
But the module  torch.nn.UnFlatten has been added to the list in the commit 9c2d4a126ef33f992e246d4eeacbb0cd803c1a5f